pkg/querier/distributor_queryable_streaming.go

```diff
@@ -67,6 +82,13 @@ func (s *streamingChunkSeries) Iterator(it chunkenc.Iterator) chunkenc.Iterator
 		chunkBytes += c.Size()
 	}
 
+	if s.chunkInfo != nil {
+		needPrint := s.chunkInfo.EndSeries()
+		if s.lastOne || needPrint {
```

Comment:
Rather than having these checks, could we just log a separate line per series? Or would that generate too much output?

Reply:
too much output as both of us experienced